lib/tizen_builder.dart

```diff
@@ -208,18 +208,31 @@ class TizenBuilder {
 
   /// Update tizen-manifest.xml with the new build info if needed.
   static void _updateManifest(TizenProject project, TizenBuildInfo buildInfo) {
-    final TizenManifest manifest =
-        TizenManifest.parseFromXml(project.manifestFile);
-    final String buildName =
-        buildInfo.buildInfo.buildName ?? project.parent.manifest.buildName;
+    if (project.isMultiApp) {
+      //UI
+      _updateManifestFile(
+          project.parent.manifest.buildName, buildInfo, project.uiManifestFile);
+      //service
+      _updateManifestFile(project.parent.manifest.buildName, buildInfo,
+          project.serviceManifestFile);
+    } else {
+      _updateManifestFile(
+          project.parent.manifest.buildName, buildInfo, project.manifestFile);
+    }
+  }
+
+  static void _updateManifestFile(
+      String projectBuildName, TizenBuildInfo buildInfo, File manifestFile) {
```

Comment:
Dart supports inner functions. Moving this function inside `_updateManifest` (and removing parameters `projectBuildName` and `buildInfo`) will make the code look cleaner.